d3viz.html

```diff
@@ -153,7 +210,11 @@
         function to_stream_key(d) {
           return d['ta'] + '-' + d['ra'];
         }
-
+/*
```

Comment:
Generally it's better to delete code instead of comment it out.  The old code is always available from the git history if you need it.